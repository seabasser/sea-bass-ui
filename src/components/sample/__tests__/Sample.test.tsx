import React from 'react';
import { render } from '@testing-library/react';
import Sample from '../Sample';

describe('<Sample /> spec', () => {
  it('Renders normally', () => {
    const { asFragment } = render(<Sample>test</Sample>);
    expect(asFragment()).toMatchSnapshot();
  });

  it('Renders with className', () => {
    const { asFragment, getByText } = render(<Sample className='testClass'>test</Sample>);
    expect(getByText('test')).toHaveClass('testClass');
    expect(asFragment()).toMatchSnapshot();
  });
});