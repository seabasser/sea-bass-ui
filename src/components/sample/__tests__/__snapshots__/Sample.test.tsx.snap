// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sample /> spec Renders normally 1`] = `
<DocumentFragment>
  <span
    class="sample"
  >
     test 
  </span>
</DocumentFragment>
`;

exports[`<Sample /> spec Renders with className 1`] = `
<DocumentFragment>
  <span
    class="sample testClass"
  >
     test 
  </span>
</DocumentFragment>
`;
